### ISSUE TITLE
Parallel Coordinate Plot

### ISSUE THREAD

**Issue by Kactaceapengu** [wandb/wandb-workspaces#46]
Hi!

I am trying to python code an Parallel Coordinate Plot, but I can't access config parameters.
Here is my code:

import wandb
import wandb_workspaces.reports.v2 as wr

# Define columns for the plot
columns = [
    # Config parameter: data.delay_size
    wr.ParallelCoordinatesPlotColumn(
        metric=wr.Config("config:delay_size.value"),
        display_name="Delay Size"
    ),
    # Config parameter: model.E_layer_dims (assuming it's nested under 'model')
    wr.ParallelCoordinatesPlotColumn(
        metric="config:learning_rate.value",
        display_name="Learning Rate"
    ),
    # Config parameter: learning_rate (top-level key)
    wr.ParallelCoordinatesPlotColumn(
        metric="c::num_epochs",
        display_name="Num Epochs"
    ),
    # Example metric logged during training
    wr.ParallelCoordinatesPlotColumn(
        metric="summary:avg_combined_test_loss",
        display_name="Test Loss"
    ),
]

# Specify the runs to include
runset = wr.Runset(project="KOOPOMICS")  # Replace with your project name

# Create the parallel coordinates plot
parallel_plot = wr.ParallelCoordinatesPlot(
    columns=columns,
    title="Parallel Coordinates Plot"
)

# Set up the panel grid
panel_grid = wr.PanelGrid(
    panels=[parallel_plot],
    runsets=[runset]
)

# Create and save the report
report = wr.Report(
    project="KOOPOMICS",  # Replace with your project name
    title="Parallel Coordinates Plot Report",
    blocks=[panel_grid]
)
report.save()

I tried to set the name inside a Config() class, but this doesn't work too.

That's what I get - which can only be correct by setting the params manually.
![Image](https://github.com/user-attachments/assets/5b9599f9-c75e-4d31-9dc6-6363473eb387)
